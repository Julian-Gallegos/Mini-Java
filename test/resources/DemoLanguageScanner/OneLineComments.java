a = b + c;
// testing
display a;