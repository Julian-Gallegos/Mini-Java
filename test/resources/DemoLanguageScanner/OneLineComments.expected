ID(a) BECOMES ID(b) PLUS ID(c) SEMICOLON ID(display) ID(a) SEMICOLON